cmd/secret/cmd/root.go

```diff
@@ -0,0 +1,82 @@
+// Package cmd provides functions set and get with cobra library
+package cmd
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/go-itools-internship/go-secret/pkg/crypto"
+	"github.com/go-itools-internship/go-secret/pkg/io/storage"
+	"github.com/go-itools-internship/go-secret/pkg/provider"
+	"github.com/spf13/cobra"
+)
+
+type root struct {
+	key       *string
+	cipherKey *string
+	value     *string
+	path      *string
+	rootCmd   *cobra.Command
+}
+
+func (r *root) RootExecute(ctx context.Context) error {
+	err := r.rootCmd.ExecuteContext(ctx)
+	return err
+}
+
+func NewRoot() *root {
+	var rootCmd = &cobra.Command{
+		Use:   "root",
+		Short: "root includes cli commands",
+	}
+	v := rootCmd.PersistentFlags().StringP("value", "v", "", "value to provider")
+	k := rootCmd.PersistentFlags().StringP("key", "k", "", "key to provider")
+	ck := rootCmd.PersistentFlags().StringP("cipherKey", "c", "", "cipher key to provider")
+	p := rootCmd.PersistentFlags().StringP("path", "p", "file.txt", "path to provider")
```

Comment:
The same for flags: use a description that actually makes use understand what are they for